samples/19.customization-plain-ui/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "sample-customization-plain-ui",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": "https://microsoft.github.io/BotFramework-WebChat/19.customization-plain-ui/",
+  "dependencies": {
+    "botframework-webchat": "*",
+    "botframework-webchat-component": "^*",
```

Comment:
What's the difference between `^*` and `*`?

Reply:
Bug in lerna I guess. We should use `*`.

Reply:
Fixed.